_js/esteemed.js

```diff
@@ -52,35 +52,46 @@ $(window).ready(function() {
     $('#main-navigation').toggle();
   });
 });
+
 // Add multi-MC forms to one page.
-$('.main-form').submit(function (e) {
-  var $this = $(this);
-  jQuery.ajax({
-    type: 'GET',
-    url: 'https://esteemed.us10.list-manage.com/subscribe/post-json?c=?',
-    data: $this.serialize(),
-    dataType: 'json',
-    contentType: 'application/json; charset=utf-8',
-    error: function (err) {
-      $('#mce-error-response-main').text('Could not connect to the server.').show();
-    },
-    success: function (data) {
-      var $required = $('.main-form-required');
-      if (data.result !== 'success') {
-        if ($required.val().length === 0) {
-          $required.addClass('mce_inline_error');
-          $required.after('<div class="mce_inline_error" id="mce_required_error">This field is required.</div>');
-        }
+function submitContactForm(token) {
+  var formData = $('#mc-embedded-subscribe-form').serialize();
+  $.post('https://esteemed-api-97dnt.ondigitalocean.app/check-human', formData)
+    .done(function(data) {
+      if (data === 'human') {
+        jQuery.ajax({
+          type: 'GET',
+          url: 'https://esteemed.us10.list-manage.com/subscribe/post-json?c=?',
```

Comment:
The forms submit to slightly varying URLs-
https://github.com/EsteemedIO/esteemedio.github.io/blob/develop/_includes/footer.html#L8
id: 4f1b3ddc37
https://github.com/EsteemedIO/esteemedio.github.io/blob/develop/_includes/footer.html#L88
id: e261e04fb2
https://github.com/EsteemedIO/esteemedio.github.io/blob/develop/_includes/footer.html#L164
id: b319d06ceb

However, we're currently submitting to the generalized URL `https://esteemed.us10.list-manage.com/subscribe/post-json?c=?`. Will this pose a problem @alex-novak ?

Reply:
@alex-novak reminder on this point